packages/salesforcedx-aura-language-server/tsconfig.json

```diff
@@ -2,12 +2,8 @@
   "extends": "../../tsconfig.common.json",
   "compilerOptions": {
     "outDir": "./out",
-    "target": "ES2022",
-    "lib": ["es2022", "dom"],
-    "noImplicitAny": false,
+    "noImplicitAny": false, // needed for ternAura.ts & ternServer.ts to avoid type errors
```

Comment:
if it's only a few places, it's better to have the rule inline-disabled or file-level disabled so that we don't have `any` stuff introduced elsewhere